CONTRIBUTING.rst

```diff
@@ -167,3 +167,46 @@ To speed up the integration process, it helps if on GitHub you `allow
 maintainers to edit your pull request
 <https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/>`_
 so they can fix small issues autonomously.
+
+.. rubric: Review guidelines
+
+A pull request will probably be reviewed by someone in a different location,
+maybe even a different time zone. Therefore, avoiding blockings and
+misunderstandings is key. The following ideas could help ease and speed
+up the review process:
+
+- Be clear when something is ready to be reviewed (see `WIP` above).
+- If a PR requires changes, say so explicitly, but politely (use emojis!).
+- When something is ready to be merged say it explicitly. If the reviewer does
+not have permission to merge, (s)he should ping the corresponding person.
+- When reviewing, rate your comments. This would help to understand if
+a comment must be addressed (e.g. it has substantial implications) or it is
+something the reviewer thought of but is a matter of "take it or leave it". A
+proposed scale is:
+  1. Comment: a change worth mentioning, a doubt or question (comment).
```

Comment:
IMHO the categories 'Comment' and 'Minor' are currently very similar and could prove slightly confusing to pick between them.
The reasoning behind this would be that: the resulting action is the same since in both cases a change is proposed that is not necessary to be applied for the PR to be merged.
The solution for this could be to remove the option of suggesting a change through a Comment.
```suggestion
  1. Comment:  a doubt or question (comment).
```
In this case, this would mean that actually this is a 'Question' category as it serves solely to provide more information to the reviewer.
So then maybe it could look like:
```suggestion
  1. Question: a doubt or question (comment).
```